libraries/bezier-rs/src/bezier/core.rs

```diff
@@ -201,6 +201,14 @@ impl Bezier {
 
 		self_points.len() == other_points.len() && self_points.into_iter().zip(other_points.into_iter()).all(|(a, b)| a.abs_diff_eq(b, max_abs_diff))
 	}
+
+	/// Returns true if the start, end and handles of the Bezier are all at the same location
+	pub fn is_single_point(&self) -> bool {
```

Comment:
Tiny nit: if 'point' is singular anyway, saying 'single point' feels redundant. Could we change this function name to `is_point`?